scene/gui/rich_text_label.cpp

```diff
@@ -901,13 +901,13 @@ int RichTextLabel::_draw_line(ItemFrame *p_frame, int p_line, const Vector2 &p_o
 				} else if (item_fx->type == ITEM_WAVE) {
 					ItemWave *item_wave = static_cast<ItemWave *>(item_fx);
 
-					double value = Math::sin(item_wave->frequency * item_wave->elapsed_time + ((p_ofs.x + off.x) / 50)) * (item_wave->amplitude / 10.0f);
+					double value = Math::sin((uint64_t)item_wave->frequency * item_wave->elapsed_time + ((p_ofs.x + off.x) / 50)) * (item_wave->amplitude / 10.0f);
```

Comment:
This change makes no sense. All values are `float` here, if it's giving precision loss warnings, it should be cast to `double` not `int`. Same for the rest of `rich_text_label.cpp`.